google_shopping.rb

```diff
@@ -7,9 +7,45 @@
 #snake case word_word not camel case wordWord
 google_data = JSON.parse(file)
 
+data = google_data['items']
+backorder = []
+images = []
+count = 0
+canon_product = []
+ebay_product = []
+
+for i in (0...data.length) do
+  if data[i]['kind'] === 'shopping#product'
+    count+= 1
+  end
+  if data[i]['product']['inventories'][0]['availability'].downcase === 'backorder'
+    backorder.push(i) 
+  end
+    if data[i]['product']['images'].length > 1 
+    images.push(i) 
+  end
+  if data[i]['product']['brand'].downcase === 'canon'
```

Comment:
You could alternatively use ```&&``` here to avoid a nested if statement, like this:
```
if data[i]['product']['brand'].downcase === 'canon' && data[i]['product']['author']['name'].include?('eBay')
```